### PR TITLE
Issue/16067: PFF Updates: Thematic map updates for 2020 Census

### LINKED ISSUE #1129: Issue/16067 PFF Updates: Thematic Map Updates (2020 Census)
### Ticket 
[Issue 16067](https://dev.azure.com/NYCPlanning/ITD/_workitems/edit/16067)
[Workflow branch strategy](https://github.com/NYCPlanning/labs-factfinder/issues/1134)
#### Thematic Maps for 2020 Census data: 
 - **ADD**  
    - [x] 65 years and over (Pop65pl)—will appear after ‘Under 18’ 
      -  10,000 or more 
      -  8,000 to 9,999 
      - 6,000 to 7,999 
      -  4,000 to 5,999 
      -  Under 4,000 
#### Thematic Maps for 2018-2022 ACS Data: 
 - **REMOVE**  
    - [x] 65 years and over (Pop65pl1)

### PR DESCRIPTION
**Pull request by horatiorosa**
### Summary
Update thematic maps for 2020 census data and update category stops for `Pop65pl`

#### Tickets
Closes #1129

### CODE REVIEW

**Review by TangoYankee — COMMENTED** [review]
I would recommend skipping PRs to the  [issue/16067-pff](https://github.com/NYCPlanning/labs-factfinder/tree/issue/16067-pff) branch. Using a branch other-than-develop let's us move quickly without worrying about stability

File: app/choropleth-config/index.js [hunk 1]
```diff
@@ -188,6 +170,24 @@ const choroplethConfigs = [
       '#e54444',
     ],
   },
+  {
+    group: 'Census',
+    id: 'pop65pl1',
```
Comment (TylerMatteo) [inline app/choropleth-config/index.js:175]: In issue 16067, this variable is referred to as `pop65pl`, not `pop65pl1`. I think Populations variable names add the `1` to the ACS version of the variable to differentiate it from the Decennial version

**Review by TylerMatteo — CHANGES_REQUESTED** [review]

File: app/choropleth-config/index.js (hunk 1 above)
Comment (horatiorosa) [reply app/choropleth-config/index.js:175]: Gocha, I wasn't aware of that. It can be a bit confusing without prior awareness.

**Comment by horatiorosa** [comment]
merging directly to trunk branch

**Pull request closed without merging**